metadata/sql/retire_concept_arv_start_date.sql

```diff
@@ -0,0 +1,17 @@
+UPDATE concept
```

Comment:
@tmvumbi2 Please use the file 900_concepts_to_retire.csv instead of a script for this